app/models/currency.rb

```diff
@@ -0,0 +1,8 @@
+class Currency
+  def self.ordered(default_currency = "USD")
+    currencies = Dictionaries.to_h[:currencies].sort
+    i = currencies.index(default_currency)
+    currencies[0], currencies[i] = currencies[i], currencies[0]
```

Comment:
@Gataniel you should not swap. You should just put default currency at first place.